packages/tools/tests/integration/components/cs-version-control-test.js

```diff
@@ -16,7 +14,12 @@ module('Integration | Component | cs version control', function(hooks) {
     // looking for its metadata, and ember-resource-metadata works
     // through ember-data's identity map.
     this.owner.register('model:thing', DS.Model.extend());
-    model = EmberObject.create({
+    // It's unfortunate we use a mock object for a full-fledged
```

Comment:
I agree, these tests are fragile, next time we're in here we should refactor to generate actual models in the right state.